.buildkite/pipeline.yml

```diff
@@ -13,9 +13,7 @@ steps:
 
     - label: ':eslint: Lint React App'
       command: make lint-js
-      plugins: &plugins
-          - $CI_TOOLKIT_PLUGIN
-          - $NVM_PLUGIN
+      plugins: *plugins
```

Comment:
This task appeared to be an outlier, where all other tasks rely upon this existing plugins anchor definition.